keps/sig-node/4800-cpumanager-split-uncorecache/README.md

```diff
@@ -289,8 +290,16 @@ N/A. This feature requires a e2e test for testing.
 #### Alpha
 
 - Feature implemented behind a feature gate flag option
-- E2E Tests will be skipped until nodes with uncore cache can be provisioned within CI hardware. Work is ongoing to add required systems (https://github.com/kubernetes/k8s.io/issues/7339). E2E testing will be required to graduate to beta.
-- Providing a metric to verify uncore cache alignment will be required to graduate to beta.
+- Test cases created for feature
```

Comment:
let's clarify that we did unit tests, added relevant metrics and added e2e tests for the metrics. It could read like:
```
- Add unit test coverage
- Added metrics to cover observability needs
- Added e2e tests for the metrics only
```